python/cugraph/structure/graph.py

```diff
@@ -1513,7 +1529,7 @@ def add_internal_vertex_id(
 class DiGraph(Graph):
     def __init__(self, m_graph=None, edge_attr=None):
```

Comment:
Maybe I'm missing something, but is the `edge_attr` arg here being used anymore?

Reply:
Removed